README.md

```diff
@@ -108,6 +108,8 @@ Documentation
 The documentation for the nRF70 BM driver is available in the `nrf70_bm_lib/docs` directory.
 To build the documentation in a Linux environment, follow the below steps:
 
+> **Note:** Currently, only Linux platforms are supported.
```

Comment:
This is fine, although, I sort of hinted it above that these instructions are for Linux

Reply:
Yes, but we still don't say others are not supported or just missing instructions :)